src/assets/scripts/client/base/DynamicPositionModel.js

```diff
@@ -359,10 +340,10 @@ export default class DynamicPositionModel {
  * @return {array}
  * @static
  */
-DynamicPositionModel.calculateRelativePosition = (coordinates, referencePosition, magneticNorth) => {
-    if (!coordinates || !referencePosition || !_isNumber(magneticNorth)) {
+DynamicPositionModel.calculateRelativePosition = (coordinates, referencePosition) => {
+    if (!coordinates || !referencePosition) {
         throw new TypeError('Invalid parameter. DynamicPositionModel.calculateRelativePosition() requires ' +
-        'coordinates, referencePosition and magneticNorth as parameters');
+        'coordinates and referencePosition and magneticNorth as parameters');
```

Comment:
you accidentally missed removing the part that I think brought you to edit this line! :laughing: 
`coordinates and referencePosition as parameters`

Reply:
You are correct

Reply:
done